universal-login-web3/lib/ui/OnboardingModal.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import {Onboarding} from '@universal-login/react';
+import React from 'react';
+
+interface OnboardingModalProps {
+  sdk: any;
+  walletService: any;
+  domains: any;
+}
+
+export function OnboardingModal({sdk, walletService, domains}: OnboardingModalProps) {
```

Comment:
We should move it to `universal-login-react` simply as `Onboarding`.
When we do so we should move styles to sass files.
#nextpr